src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -41,6 +42,10 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
+                if (!class_exists(Intl::class)) {
+                    throw new LogicException('The "symfony/intl" component is required to use the Country type.');
+                }
```

Comment:
Could we move this to `IntlCallbackChoiceLoader` instead?

Reply:
the message would be less obvious.. currently it hints `Country type` etc.